src/Entity/Pap/Campaign.php

```diff
@@ -25,6 +25,7 @@
  *     attributes={
  *         "order": {"createdAt": "DESC"},
  *         "pagination_enabled": false,
```

Comment:
what do you thing about removing `"pagination_enable": false` ?
Api platform will handle the pagination for collection operation. so need to use `"pagination_client_enabled": true`

Reply:
Done.